uniffi/src/interface/mod.rs

```diff
@@ -363,12 +382,19 @@ impl Function {
         &self.ffi_func
     }
 
+    pub fn throws(&self) -> Option<&str> {
+        self.attributes.get_throws_err()
+    }
+
     fn derive_ffi_func(&mut self, ci_prefix: &str) -> Result<()> {
         self.ffi_func.name.push_str(ci_prefix);
         self.ffi_func.name.push_str("_");
         self.ffi_func.name.push_str(&self.name);
         self.ffi_func.arguments = self.arguments.clone();
         self.ffi_func.return_type = self.return_type.clone();
+        // Theoritically this should always be true
+        // but it's this way until we implement handling for panics
+        self.ffi_func.has_out_err = self.throws().is_some();
```

Comment:
> Or do you think we'll want all our Swift functions to be marked as throws, and have the caller use try! explicitly?

I was thinking this - which is why I kept all functions as `throws` for now in swift - but now that I think about it, might be better to do the `try!` approach 🤔 Seems to better reflect the fact that the errors are not explicit (i.e if we panic in Rust, maybe it's better to crash swift etc..) either way, I feel like this will surface in #32, for now, all functions have `throws` in swift.